pipelines/rj_pic__disparos_email/template_engine.py

```
@@ -0,0 +1,53 @@
+        template_dir = self.template_path.parent
+        self.env = Environment(
+            loader=FileSystemLoader(str(template_dir)),
+            autoescape=True  # Escapa HTML automaticamente para segurança
+        )
+
+        # Carrega o template
+        template_name = self.template_path.name
+        try:
+            self.template = self.env.get_template(template_name)
+        except TemplateNotFound:
+            raise FileNotFoundError(f"Template não encontrado: {template_name}")
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)